app/views/components/pageflow/admin/users_tab.rb

```diff
@@ -3,16 +3,13 @@ module Admin
     class UsersTab < ViewComponent
       def build(theming)
         account = theming.account
-        if account.users.any?
-          table_for account.users, :i18n => User do
-            column :full_name do |user|
-              link_to user.full_name, admin_user_path(user)
-            end
-          end
-        else
-          div :class => "blank_slate_container" do
-            span :class => "blank_slate" do
-              I18n.t('pageflow.admin.accounts.no_members')
+        embedded_index_table account.users.order(
+            params[:order] && User.column_names.include?(params[:order].gsub('_asc', '').gsub('_desc', '')) ?
```

Comment:
Line is too long. [109/100]<br>Indent the first parameter one step more than the start of the previous line.<br>Avoid multi-line ?: (the ternary operator); use `if`/`unless` instead.